servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilder.java

```diff
@@ -368,7 +368,7 @@ private static StreamingHttpClientFilterFactory appendFilter(
                 new RetryingServiceDiscoverer<>(new StatusAwareServiceDiscoverer<>(serviceDiscoverer, sdStatus),
                         serviceDiscovererRetryStrategy == null ?
                                 DefaultServiceDiscoveryRetryStrategy.Builder.<R>withDefaults(exec.executor(),
-                                        ofSeconds(60)).build()
+                                        ofSeconds(10), ofSeconds(5)).build()
```

Comment:
Consider creating 2 plg-private constants that we can share between partitioned and single-address clients to make sure the values are always consistent.